<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventaire</class>
 <widget class="QMainWindow" name="Inventaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventaire</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableStock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>1131</width>
      <height>491</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="French" country="Switzerland"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marque</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modèle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N/S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Taille</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Etat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Coût</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Retour</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnFiltrer">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBrand">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Marque</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textModel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Modèle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSerialNumber">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>N/S</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textState">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrer par :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter des articles</string>
    </property>
   </widget>
   <zorder>btnFiltrer</zorder>
   <zorder>tableStock</zorder>
   <zorder>textBrand</zorder>
   <zorder>textModel</zorder>
   <zorder>textSerialNumber</zorder>
   <zorder>textState</zorder>
   <zorder>label</zorder>
   <zorder>btnAdd</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableStock</tabstop>
  <tabstop>textBrand</tabstop>
  <tabstop>textModel</tabstop>
  <tabstop>textSerialNumber</tabstop>
  <tabstop>textState</tabstop>
  <tabstop>btnFiltrer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
